### DISCUSSION TITLE
Gitの使い方

### DISCUSSION THREAD

**Discussion by PrefKarafuto** [PrefKarafuto/ex0ch#15] [Q&A]
新規参入の方向けにgitコマンド/GitHubの使い方、難しい・躓いた点や対処方法などをまとめていきましょう。

**Comment by PrefKarafuto** [comment]
資料などを集積しましょう。
https://qiita.com/konweb/items/621722f67fdd8f86a017
https://qiita.com/atsushi101011/items/96054ebbd876e378cac4
https://www.sejuku.net/blog/71003

**Reply by koushinkyouto** [reply]
Gitの導入方法(Windows)
https://prog-8.com/docs/git-env-win

**Comment by Kusaimara** [comment]
要課金みたいですがcodespacesなる機能があって、外部の仮想環境でスクリプトを動かせる機能があるようです。
https://docs.github.com/ja/codespaces/overview

Dockerを用いた使い方など
https://info.drobe.co.jp/blog/engineering/github-codespaces
https://zenn.dev/yuiseki/articles/89b62b1cfdfaf0
https://tech.anti-pattern.co.jp/github-codespaces/
